### PR TITLE
fix: clean up CodeQL alerts

### PR DESCRIPTION
**Pull request by wochinge**
## Summary
- fully escape markdown table cells before rendering PR comments
- exclude scripts/ from CodeQL analysis alongside dist/
- use a fresh private temp directory for Node SDK installs instead of a predictable shared path
- rebuild dist for runtime changes

## Checks
- pnpm vitest run tests/comment.test.ts
- pnpm run lint
- pnpm run typecheck
- pnpm vitest run tests/wrappers.test.ts tests/comment.test.ts
- pnpm run build
- pre-commit hook ran: build, lint, typecheck, format:check, check:schema

### CODE REVIEW

File: src/comment.ts [hunk 1]
```diff
@@ -105,7 +105,10 @@ function stringifyCell(v: unknown): string {
 /** Escape + truncate a value to fit inside a markdown table cell. */
 function cell(v: unknown, maxLen = CELL_MAX): string {
   let s = stringifyCell(v);
-  s = s.replace(/\|/g, "\\|").replace(/\r?\n/g, " ");
+  s = s
+    .replace(/\\/g, "\\\\")
+    .replace(/\|/g, "\\|")
+    .replace(/[\r\n]+/g, " ");
   if (s.length > maxLen) s = s.slice(0, maxLen - 1) + "…";
   return s || "—";
```
[the remark this answers is not in the document]
Comment (wochinge) [reply src/comment.ts]: Good catch — addressed in ddbee93. `cell()` now normalizes/truncates before applying markdown escapes, so slicing cannot split a generated `\\` / `\|` escape sequence. I also added a focused regression test with a pipe at the truncation boundary.

**Comment by wochinge** [comment]
@claude review
